spec/requests/request_throttling_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'throttling for xml requests' do
+
+  let(:limit) { 30 }
```

Comment:
I'm concerned about how long this test will take to run. Could we stub the limit to like 3 or 4 just for the test case?